repos/system_upgrade/el7toel8/actors/repositoriesblacklist/libraries/repositoriesblacklist.py

```diff
@@ -1,13 +1,21 @@
 from leapp import reporting
-from leapp.libraries.common.config import get_product_type
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common.config.version import get_source_major_version, get_target_major_version
 from leapp.libraries.stdlib import api
 from leapp.models import (
     CustomTargetRepository,
     RepositoriesBlacklisted,
     RepositoriesFacts,
-    RepositoriesMap,
+    RepositoriesMapping
 )
 
+# {OS_MAJOR_VERSION: PESID}
+UNSUPPORTED_PESIDS = {
```

Comment:
Are the PES IDs unsupported or the underlying repositories?

Reply:
both. the PESID represent family of repositories. In shis case the unsupported repoids are in a unsuuported PES id family.